kibble/config/config_test.go

```diff
@@ -41,3 +39,9 @@ func TestLoadLanguagesConfigWithObjects(t *testing.T) {
 	assert.Equal(t, cfg.Languages["it"].Code, "it_IT")
 	assert.Equal(t, cfg.Languages["it"].Name, "Italian")
 }
+
+func TestLoadConfig(t *testing.T) {
+	sitePath = "../sample_site/kibble.json"
+	cfg := LoadConfig(false, "xxx", false)
+	assert.Equal(t, true, cfg.RemoteTranslations)
```

Comment:
Don't forget we need tests to hit the AC's for the negative cases - false and missing. We probably need independent `kibble.json` for each and then you should be able to set the `sitePath` for each test.